Chapters/ThreePillarsOfSpec/ThreePillarsOfSpec.md

```diff
@@ -72,207 +72,182 @@ The following snippet creates a window with the list of methods of the class `Po
-In addition to these primitive methods, the basic widgets provide more specific hooks, e.g., when an item in a list is selected (`whenSelectionChangedDo:`). 
+Value holders provide the method `whenChangedDo:` that is used to register a block to perform on change and the method `whenChangedSend: aSelector to: aReceiver` to send a message to a given object. In addition to these primitive methods, the basic widgets provide more specific hooks, e.g., when an item in a list is selected (`whenSelectionChangedDo:`).
 
-### Defining UI Layouts 
+### Defining UI Layouts
 
-@sec_layoutmethod 
+@sec_layoutmethod
 
-Widget layout is defined by specifying methods that state how the different widgets that compose a UI are placed. 
-In addition, it also specifies how a widget reacts when the window is resized. 
-As we will see later, these methods can have different names. 
+Widget layout is defined by specifying methods that state how the different widgets that compose a UI are placed. In addition, it also specifies how a widget reacts when the window is resized. As we will see later, these methods can have different names.
 
 
 The method `defaultLayout` is an instance method, but it can be also defined at the class level. Put differently, typically all the instances of the same user interface have the same layout but a layout can be specific to one instance and be dynamic.
 
-**Note.** Specifying a layout is mandatory, as without it the UI would show no widgets to the user. 
+**Note.** Specifying a layout is mandatory, as without it the UI would show no widgets to the user.
 
-#### Using setter `layout:` message
+#### Using setter message `layout:`
 
 We recommend to clearly separate presenter initialization (`initializePresenters` and `defaultLayout`). You can, however, also use the `layout:` message to set a layout during the presenter initialization phase.
 
 
-#### Having multiple layouts for a widget 
- 
-For the same UI, multiple layouts can be described, and when the UI is built the use of a specific layout can be indicated. To do this, instead of calling `open` (as we have done until now), use the `openWithLayout:` message with a layout as an argument.
-
+#### Multiple layouts for a widget
 
+For the same UI, multiple layouts can be described, and when the UI is built the use of a specific layout can be indicated. To do this, instead of calling `open` (as we have done until now), use the `openWithLayout:` message with a layout as an argument.
 
 
 
- 
-### Conclusion 
+### Conclusion
 
-In this chapter, we have given a more detailed description of how the three fundamental methods of Spec: `initializePresenters`, `defaultLayout`, and `connectPresenters` are each responsible for a different aspect of the user interface building process. We also discussed in detail the ability to use different layout methods and how the lookup of layout methods is performed. 
```

Comment:
`We also discussed in detail the ability to use different layout methods and how the lookup of layout methods is performed.` is not correct. `in detail` is not true, and `how the lookup of layout methods is performed` is not covered. Therefore I removed this sentence.